src/ActionList/Item.tsx

```diff
@@ -70,13 +112,12 @@ const StyledItem = styled.div<{variant: ItemProps['variant']} & SxProp>`
   padding: 6px ${get('space.2')};
   display: flex;
   border-radius: ${get('radii.2')};
-  color: ${({variant}) => (variant === 'danger' ? get('colors.text.danger') : 'inherit')};
+  color: ${({variant, item}) => getItemVariant(variant, item?.disabled).color};
 
   @media (hover: hover) and (pointer: fine) {
     :hover {
-      background: ${props =>
-        props.variant === 'danger' ? get('colors.bg.danger') : get('colors.selectMenu.tapHighlight')};
-      cursor: pointer;
+      background: ${({variant, item}) => getItemVariant(variant, item?.disabled).hoverBackground};
+      cursor: ${({variant, item}) => getItemVariant(variant, item?.disabled).hoverCursor};
```

Comment:
Good catch with `cursor`! I was only thinking about the color changes. 🎊

Reply:
Related: Should we add [`pointer-events: none`](https://developer.mozilla.org/en-US/docs/Web/CSS/pointer-events) to `disabled` `Item`s?

Reply:
>Related: Should we add pointer-events: none to disabled Items?

I don't think so. A tooltip explaining why something is disabled is a pretty common pattern, and `pointer-events: none` would prevent that from being possible.